deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJobTest.java

```diff
@@ -434,7 +434,7 @@ public void resumeIngestWorkObjectTest() throws Exception {
         verify(work).setPrimaryObject(mainPid);
 
         verify(jobStatusFactory).setCompletion(eq(jobUUID), eq(2));
-        verify(jobStatusFactory).incrCompletion(eq(jobUUID), eq(1));
+        verify(jobStatusFactory, never()).incrCompletion(eq(jobUUID), eq(1));
```

Comment:
`incrCompletion` is still getting called, isn't it? Just not with the value 1

Reply:
No, but it looks like setTotalCompletion() is being set